.github/workflows/deployProd.yml

```diff
@@ -99,4 +101,53 @@ jobs:
           webhook_url: ${{ secrets.SR_ALERTS_SLACK_WEBHOOK_URL }}
           user_map: $${{ secrets.SR_ALERTS_GITHUB_SLACK_MAP }}
 
-# a post-prod health check workflow is defined in smokeTestDeployProd. See the Alert response wiki page for more details
\ No newline at end of file
+  failed_deploy_pagerduty_alert:
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.deploy.result != 'success' }}
+    needs: [ deploy ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Trigger PagerDuty alert
+        uses: ./.github/actions/pagerduty-alert
+        with:
+          summary: ":siren-gif: Deploy to ${{ env.DEPLOY_ENV }} failed. ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} :siren-gif:"
+          severity: error
+          routing_key: ${{ secrets.pagerduty_prod_integration_key }}
+
+  # See the Alert response wiki page for more details on post-deploy smoke test
+  smoke_test_front_and_back_end:
+    runs-on: ubuntu-latest
+    needs: [ deploy ]
+    if: ${{ needs.deploy.result == 'success' }}
```

Comment:
Similar to my question above, do we want to run this to double check that things are still up even if the deploy fails?